runner/src/main.rs

```diff
@@ -180,7 +182,30 @@ fn run_examples(opt: &RunExamples) -> Step {
     }
 }
 
-fn build_wasm_module(name: &str, target: &Target, example_name: &str) -> Step {
+fn run_functions_examples(opt: &RunFunctionsExamples) -> Step {
+    let examples: Vec<Example> = example_toml_files()
+        .map(|path| {
+            toml::from_str(&read_file(&path)).unwrap_or_else(|err| {
+                panic!("could not parse example manifest file {:?}: {}", path, err)
+            })
+        })
+        .filter(|example: &Example| example.example_type == ExampleType::OakFunctions)
+        .collect();
+
+    Step::Multiple {
+        name: "oak-functions examples".to_string(),
+        steps: examples
+            .iter()
+            .filter(|example| match &opt.example_name {
+                Some(example_name) => &example.name == example_name,
+                None => true,
+            })
+            .map(|example| run_functions_example(opt, example))
+            .collect(),
+    }
+}
+
+fn build_wasm_module(name: &str, target: &Target, out_dir: &str) -> Step {
```

Comment:
I'm not sure how to interpret `out_dir` here, especially since in #1981 the logic is more tightly bound to how modules actually get compiled, based on the Cargo.toml and workspace configuration. In fact it seems that `out_dir` is ignored for Rust targets now anyways?